From a7cc4c7fd1fddf215e4f393ee0058750cdda0747 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 26 Jul 2013 13:03:14 +0800
Subject: [PATCH 03/21] ENGR00272747 Use blk_device to replace device variable
 for jb4.3 upgrade

fstab_rec used to replace the Volume structure in jb4.3

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 roots.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/roots.cpp b/roots.cpp
index 9f3b82f..16885da 100644
--- a/roots.cpp
+++ b/roots.cpp
@@ -278,7 +278,7 @@ int format_volume(const char* volume) {
     if (strcmp(v->fs_type, "ubifs") == 0) {
         int ret;
         LOGW("formating by ubiVolumeFormat");
-        ret = ubiVolumeFormat(v->device);
+        ret = ubiVolumeFormat(v->blk_device);
         if (ret != 0) {
             LOGE("ubiVolumeFormat return error:%d", ret);
             return -1;
-- 
1.8.0

